src/awareness-manager.ts

```diff
@@ -68,6 +69,7 @@ export class AwarenessManager {
 		}
 
 		AwarenessManager.__instance = new AwarenessManager( awareness, await getCurrentUserInfo() );
+		SessionActivityController.initialize( awareness );
```

Comment:
This feels like an odd place to initialize it imo. I feel like it should be in the awareness manager constructor in some way? This is assuming the other comment I made about changing how SessionActivityController is initialized.